src/features.ts

```diff
@@ -32,3 +32,7 @@ export const features: FeaturesList = {
   pubsub,
   storage,
 };
+
+export interface TestFeatureList extends FeaturesList {
+  cleanup();
+}
```

Comment:
I am not super excited about the name `TestFeatureList`. I created it more as a proof of concept for the fix.